components/role-mgt/org.wso2.carbon.identity.role.mgt.core/src/main/java/org/wso2/carbon/identity/role/mgt/core/dao/RoleDAOImpl.java

```diff
@@ -1482,6 +1512,28 @@ public Set<String> getSystemRoles() {
         return systemRoles;
     }
 
+    @Override
+    public int getRolesCount(String tenantDomain) throws IdentityRoleManagementException {
+        int tenantId = IdentityTenantUtil.getTenantId(tenantDomain);
```

Comment:
Add a new line after the method signature.

Reply:
Fixed with cd84a4114fc20acf8dbd6155122c658d45aebd46